### PR TITLE
Add date column and value to custom forms

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
When a user creates a form in Activity and publishes it, they would like the table created in Track to have a submission date for each entry.

## Approach
- I appended the new column to the given ones
- I always add the date when a new data is sent to be saved

### Furter Info
Related ticket: #480

### CODE REVIEW

File: silo/api.py [hunk 1]
```diff
@@ -157,7 +161,7 @@ def create(self, request, *args, **kwargs):
         form_name = serializer.data['name']
         read_name = serializer.data['name']
         columns = serializer.data['fields']
-        columns.append({'name': 'submission_time', 'type': 'time'})
+        columns = columns + self._default_columns
```
Comment (Menda) [inline silo/api.py]: `columns +=`

File: silo/api.py [hunk 2]
```diff
@@ -261,8 +265,11 @@ def save_data(self, request):
                             status=status.HTTP_400_BAD_REQUEST)
 
         if data:
-            submission_date = datetime.now().strftime('%H:%M:%S')
-            data.update({'submission_time': submission_date})
+            # add the timestamp
+            submission_date = datetime.now().strftime('%Y-%m-%d')
```
Comment (Menda) [inline silo/api.py]:
```
now = datetime.now()
submission_date = now.strftime('%Y-%m-%d')
submission_time = now.strftime('%H:%M:%S')
```

**Comment by jefmoura** [comment]
Done @Menda

File: silo/tests/test_customformview.py [hunk 3]
```diff
@@ -130,7 +131,7 @@ def test_create_customform_success_superuser(self):
         silo_id = response.data['id']
         silo = Silo.objects.get(pk=silo_id)
         form_name = '{} - {}'.format(data['name'], wflvl1.name)
-        fields.append({'name': 'submission_time', 'type': 'time'})
+        fields = fields + CustomFormViewSet._default_columns
```
Comment (Menda) [inline silo/tests/test_customformview.py]: `fields +=`

Comment (jefmoura) [reply silo/tests/test_customformview.py]: Something else? I'd prefer to change everything at once.

**Pull request merged**